### PR TITLE
Added SceJpegEncArm enums and edited documentation.

### PR DESCRIPTION
**Pull request by bythos14**
Added missing enums, and adjusted documentation to represent SceJpegEncArm more correctly. Also changed the type of some function args.

### CODE REVIEW

File: include/psp2/jpegencarm.h [hunk 1]
```diff
@@ -1,101 +1,165 @@
...
 int sceJpegArmEncoderEncode(SceJpegArmEncoderContext context, const void *inBuffer);
 
 /**
- * Set encoder compression ratio
+ * Set the encoder compression ratio.
  *
- * @param[in] context - A pointer to an already initialized ::SceJpegArmEncoderContext
- * @param[in] ratio - A value between 0 and 255 (higher = better compression, lower = better speed)
+ * @param[in] context - An already initialized ::SceJpegArmEncoderContext.
+ * @param[in] ratio   - A value between 1 and 256 (higher = better compression, lower = better speed).
```
Comment (d3m3vilurr) [inline include/psp2/jpegencarm.h:123]: 2 to 255?

Comment (bythos14) [reply include/psp2/jpegencarm.h]: Fixed, was a mistake from early testing.

File: include/psp2/jpegencarm.h [hunk 2]
```diff
@@ -1,101 +1,165 @@
...
 #include <psp2/types.h>
-#include <psp2/jpegenc.h>
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+#define SCE_JPEGENCARM_MIN_COMP_RATIO     1   //!< Lowest compression ratio, best quality.
+#define SCE_JPEGENCARM_DEFAULT_COMP_RATIO 64  //!< Default compression ratio.
+#define SCE_JPEGENCARM_MAX_COMP_RATIO     255 //!< Highest compression ratio, lowest quality.
```
Comment (d3m3vilurr) [inline include/psp2/jpegencarm.h:22]: @bythos14 here is 1 to 255. but a document of the `ratio` is 0 to 256.
what is right value?

Comment (bythos14) [reply include/psp2/jpegencarm.h:22]: The documentation says **between** 0 and 256, so 0 and 256 are not included in that range of values.

Comment (bythos14) [reply include/psp2/jpegencarm.h:22]: So 1 - 255 is the correct range

Comment (d3m3vilurr) [reply include/psp2/jpegencarm.h:22]: in normally, `between a and b` is `a <= x <= b`. isn't it?

Comment (bythos14) [reply include/psp2/jpegencarm.h:22]: Probably. I guess it would be better to the term inclusively here. I've made the adjustments.

Comment (teakhanirons) [reply include/psp2/jpegencarm.h:22]: Documentation from where?

Comment (bythos14) [reply include/psp2/jpegencarm.h:22]: The documentation written in the header.

**Pull request merged**